eoscostarica/eosio-spend-explorer
# build(env): Update hyperion endpoint #39

### build(env): Update hyperion endpoint

### What does this PR do?

- Resolve #39 
- Resolve #32 

### Steps to test

1. Run the project, to know more about this, visit the readme file
2. Go to localhost:9695
3. Go to eden_delegates and see the data

![imagen](https://user-images.githubusercontent.com/49013556/182643091-6eb84dcd-1972-466d-8acc-2043584987d4.png)

**We build the following diagram to make the DB.**
![image](https://user-images.githubusercontent.com/56526123/182964581-6abaaaea-7d18-449c-a82c-77f7458b436f.png)

- The tables Eden_delegate and Eden_election have a one-to-many relationship.
- The tables Eden_election and Eden_transaction have a one-to-many relationship.

#### CheckList
